Client/Impl/Builder/CamundaCloudClientBuilder.cs

```diff
@@ -75,6 +75,18 @@ public ICamundaCloudClientBuilderFinalStep UseAuthServer(string url)
             return this;
         }
 
+        public ICamundaCloudClientBuilderFinalStep UsePersistedStoragePath(string path)
+        {
+            if (path is null)
+            {
+                // use default
+                return this;
+            }
```

Comment:
I would like to remove this, since we handle nulls here https://github.com/camunda-community-hub/zeebe-client-csharp/blob/main/Client/Impl/Builder/CamundaCloudTokenProviderBuilder.cs#L78-L83 and would like to fail if we put null in (likely by mistake, otherwise we would hide this and only realize later that we are not writing to the expected path)